css/main.css

```diff
@@ -0,0 +1,137 @@
+h1 {
+    color: #dedede;
+}
+
+p {
+    color: #eeeeee;
+}
+
+/* Navigation */
+
+.navbar {
+    background: rgb(255, 0, 0, 0.7) !important;
+}
+
+.nav-link,
+.navbar-brand {
+    color: #dedede !important;
+    cursor: pointer !important;
+}
+
+.nav-link {
+    margin-right: 1em !important;
+}
+
+.nav-link:hover {
+    background: rgba(0, 0, 0, 0);
+    color: #af0404 !important;
+}
+
+.navbar-collapse {
+    justify-content: flex-end;
+}
+
+.navbar-toggler {
+    background: rgb(255, 0, 0, 0.7) !important;
+}
+
+.header {
+    background-image: url('../images/bkgtop.png');
+    background-attachment: fixed;
+    background-size: cover;
+    background-position: center;
+    width: 100%;
+    height: 850px;
```

Comment:
Don't forget to use rem!

You can use the 62.5% hack as I explained you before.

```css
html {
  /* 16px * 62.5% = 10px */
  font-size: 62.5%;
}
body {
  /* 1rem = 10px */
  font-size: 1.6rem;
}
```

```css
.header {
   height: 85rem; /* 850px */
}
```

Change everywhere from px to rem :)

Reply:
fixed it. 
but some reason would not work if I tried to apply both at same time,
worked only when I put in
body {
  /* 1rem = 10px */
  font-size: 1.6rem;
}

Reply:
Can you do it and commit the change, so I can see on my side what's wrong?